5.5/contrib/httpdconf.sed

```diff
@@ -0,0 +1,7 @@
+s/^Listen 80/Listen 0.0.0.0:8080/
+s/^User apache/User default/
+s/^Group apache/Group default/
+s%^DocumentRoot "/opt/rh/httpd24/root/var/www/html"%DocumentRoot "/opt/openshift/src"%
+s%^<Directory "/opt/rh/httpd24/root/var/html"%<Directory "/opt/openshift/src"%
+s%^ErrorLog "logs/error_log"%ErrorLog "/tmp/error_log"%
+s%CustomLog "logs/access_log"%CustomLog "/tmp/access_log"%
```

Comment:
actually i guess let's not, for now..but we probably have to decide what to do about it at some point.